DDDEastAnglia/Controllers/HomeController.cs

```diff
@@ -9,15 +9,17 @@ namespace DDDEastAnglia.Controllers
     public class HomeController : Controller
     {
         private readonly IConferenceLoader conferenceLoader;
+        private readonly SponsorModelQuery sponsorModelQuery;
 
-        public HomeController(IConferenceLoader conferenceLoader)
+        public HomeController(IConferenceLoader conferenceLoader, SponsorModelQuery sponsorModelQuery)
```

Comment:
I feel as though there should be an interface that this implements - `IQuery<T>`, perhaps? Maybe this is something to add when we have more than one query object, though :smile: